idaes/power_generation/carbon_capture/compression_system/compressor.py

```diff
@@ -417,113 +339,81 @@ def delta_enth_polytropic_con(b, t):
-            Pratio = b.ratioP[t]
+            k_v = self.control_volume.properties_in[t].heat_capacity_ratio
+            a = (k_v - 1) / k_v
+            Pd = self.vapor_outlet.pressure[t]
+            Ps = self.vapor_inlet.pressure[t]
             Tin = properties_in[t].temperature
             mw = properties_in[t].mw
             gas_const = const.gas_constant
-            return b.z_s * (gas_const / mw) * Tin * (1 / a) * (Pratio**a - 1)
+            return b.z_s * (gas_const / mw) * Tin * (1 / a) * ((Pd/Ps)**a - 1)
 
+        # Dimensionless Isentropic Head Coefficient (Performance Curve)
         @self.Constraint(self.flowsheet().config.time,
                          doc="Dimensionless Isentropic Head Coefficient")
         def psi_s_eqn(b, t):
             return b.psi_s[t] == 2 * b.ys_model[t] / (b.U2[t]**2)
 
-        # Ang is used to estimate coeffcients a, b, c for first stage
-        if self.config.first_stage is True:
-            self.Ang = Var(self.flowsheet().config.time,
-                           initialize=0,
-                           doc="Inlet Guide Vanes Angle",
-                           bounds=(-15, 90), units=pyunits.dimensionless)
-
-            @self.Expression(self.flowsheet().config.time)
-            def coeff_a(b, t):
-                return -38.46 * exp(-0.009058 * b.Ang[t]) + (
-                    - 0.1043) * exp(0.09821 * b.Ang[t])
-
-            @self.Expression(self.flowsheet().config.time)
-            def coeff_b(b, t):
-                return 6.863 * exp(-0.01329 * b.Ang[t]) \
-                    + 0.02018 * exp(0.09443 * b.Ang[t])
-
-            @self.Expression(self.flowsheet().config.time)
-            def coeff_c(b, t):
-                return -0.00891 * exp(0.06757 * b.Ang[t]) \
-                    + 0.9661 * exp(0.006228 * b.Ang[t])
-        else:
-            self.coeff_a = Param(default=-428.64,
-                                 mutable=True,
-                                 doc="Performance Curve Parameter A")
-            self.coeff_b = Param(default=29.178,
-                                 mutable=True,
-                                 doc="Perforamance Curve Parameter B")
-            self.coeff_c = Param(default=0.9737,
-                                 mutable=True,
-                                 doc="Performance Curve parameter C")
-
-        @self.Constraint(self.flowsheet().config.time,
-                         doc="Correlation between psi_s and psi_3")
-        def psi_s_stage_eqn(b, t):
-            if self.config.first_stage is True:
-                return b.psi_s[t] == b.coeff_a[t] * b.psi_3[t]**2 + \
-                    b.coeff_b[t] * b.psi_3[t] + b.coeff_c[t]
-            else:
-                return b.psi_s[t] == b.coeff_a * b.psi_3[t]**2 + \
-                    b.coeff_b * b.psi_3[t] + b.coeff_c
-
-    def initialize(blk, state_args={}, outlvl=idaeslog.NOTSET, solver='ipopt',
+    def initialize(self, state_args={}, outlvl=idaeslog.NOTSET, solver='ipopt',
```

Comment:
A recent PR changed to standard approach for solver setting in initialization routines to fix some bugs in the general approach.

The standard arguments should now be `solver=None, optarg={}`

Later comments will show how to make this work.